packages/math-input/src/components/fraction-keypad.tsx

```diff
@@ -15,16 +15,20 @@ import Keypad from "./keypad";
 import Styles from "./styles";
 import TouchableKeypadButton from "./touchable-keypad-button";
 
+import type {State} from "../store/types";
 import type {CursorContext} from "./input/cursor-contexts";
 
 const {row, roundedTopLeft, roundedTopRight} = Styles;
 
-type Props = {
-    cursorContext: CursorContext;
+interface ReduxProps {
+    cursorContext?: CursorContext;
     dynamicJumpOut: boolean;
+}
+
+interface Props extends ReduxProps {
     roundTopLeft: boolean;
     roundTopRight: boolean;
-};
+}
```

Comment:
Thanks for splitting up these props.  Please be cautious with `interfaces` which multiple definitions of the same interface will get merged.  This is especially scary with things like `Node` which is a globally defined interface.

Reply:
I was surprised to see `interface` also, but that's how the docs suggest to do it: https://react-redux.js.org/using-react-redux/usage-with-typescript#inferring-the-connected-props-automatically